recipes/libnl-devel-cos6-x86_64/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: libnl-devel-cos6-x86_64
+  version: 1.1.4
+
+source:
+  - url: http://mirror.centos.org/centos/6.10/os/x86_64/Packages/libnl-devel-1.1.4-2.el6.x86_64.rpm
+    sha256: 6c1749d9f8e9cbe31c84067a7da5f9861c2cd61a0df9b83107b6d045f24a766a
+    folder: binary
+  - url: http://vault.centos.org/6.10/os/Source/SPackages/libnl-1.1.4-2.el6.src.rpm
+    folder: source
```

Comment:
Currently there is no good way to determine the checksum of the source RPM. Please see issue ( https://github.com/conda/conda-build/issues/3552 ) for details.

Reply:
Can't you add it manually? That's what most packages do except in PyPI where sha256 are published.

Reply:
Could we move this discussion to the linked issue?

Reply:
I'm sorry but that's a slightly different issue.
For eg: say there's a project that doesn't provide SHA256, do we just create a conda-build issue or do we ask the recipe author to generate one?

Reply:
The thing is we are going to add a lot of CDTs. I'm somewhat reluctant to have the same conversation across several PRs. Let's try to consolidate the discussion in that issue to make it easier to follow (even if you consider it somewhat off-topic). Sound good? 🙂

Reply:
I'm okay with that as long as these PRs are not merged.

Reply:
I'd be ok waiting here if there is active discussion on that issue. Though I still haven't seen this though 😉

Reply:
I've gone ahead and dropped the source RPM as it seems to work without it. Hopefully that addresses this concern.